edx_rest_framework_extensions/tests/test_permissions.py

```diff
@@ -64,19 +63,39 @@ def test_has_permission(self, has_user_in_url):
 
 
 @ddt.ddt
-class IsJwtAuthenticatedTests(TestCase):
-    """ Tests for the IsJwtAuthenticated permission class. """
+class JwtRestrictedApplicationTests(TestCase):
```

Comment:
Is `JwtApplicationTests` too meaningless?

Reply:
I renamed it to `JwtApplicationPermissionsTests`.